src/components/pages/HomePage.jsx

```diff
@@ -1,28 +1,160 @@
+import mainImage from "../../assets/imgMainPage.jpg";
+import GreenBlobImage from "../../assets/blob_green.jpg";
+import YellowBlobImage from "../../assets/blob_yellow.jpg";
+import PurpleBlobImage from "../../assets/blob_dusty_purple.jpg";
+import Music from "../../assets/icons/icons-musical.png";
+import Acting from "../../assets/icons/icons8-acting.png";
+import Storytelling from "../../assets/icons/icons8-storytelling.png";
+import Arts from "../../assets/icons/icons-art.png";
+import Photography from "../../assets/icons/icons8-photography.png";
+import Games from "../../assets/icons/icons-chess.png";
+import Films from "../../assets/icons/icons8-film.png";
+import Animation from "../../assets/icons/icons8-3d.png";
+import Handicraft from "../../assets/icons/icons8-hand-holding.png";
+import ArrowUp from "../../assets/icons/icons-arrow-up.png";
+import ArrowSearch from "../../assets/icons/icon-search.svg";
 
 const HomePage = () => {
-
   const navigate = useNavigate();
+
   const search = () => {
     navigate("/search");
   };
-  const login = () => {
-    navigate("/login");
-  };
-  const register = () => {
-    navigate("/register");
-  };
-
 
   return (
-    <>
-      <h1 className="text-3xl font-spartan font-bold">HOME PAGE</h1>
-      <button onClick={search}>Search classes</button>
-      <button onClick={login}>Log In</button>
-      <button className='bg-red hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-lg py-1 px-7 rounded-lg' 
-              onClick={register}>Join</button>
-    </>
+    <div className="flex flex-col my-auto lg:mx-auto lg:px-8 sm:mx-10 sm:my-12">
+      <section className="flex flex-col lg:flex-row lg:justify-between justify-center sm:pl-2">
+        <div className="lg:pt-36 pt-4 sm:pt-10 lg:text-left text-center items-center justify-center">
+          <h1 className="text-4xl sm:text-5xl md:text-6xl font-spartan font-semibold tracking-wide leading-tight pt-3 mb-8">
+            Helping unlock
+            <br />
+            your talent with
+            <br />
+            the best teachers
+          </h1>
+          <div className="flex justify-center lg:justify-start">
+          <button
+            className="flex items-center bg-red hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-2xl py-4 px-8 rounded-lg"
+            onClick={search}
+          >
+            Get started
+            <img src={ArrowSearch} alt="Arrow Right" className="ml-4 w-6 h-6" />
+          </button>
+          </div>
+         
+        </div>
+
+        <div className="m-10">
+          <img className="w-[700px]" src={mainImage} alt="art skills image" />
+        </div>
+      </section>
+      <section className="flex flex-col lg:flex-row items-center md:m-auto justify-center pt-2 lg:pt-16">
+        <div className="relative w-full lg:max-w-lg md:pb-12 pl-0">
```

Comment:
I believe it's better to use the small blobs not as images inside the div but as background image. And maybe add bg-cover 

<div class="bg-[url('/img.png')]">
  <!-- ... -->
</div>
or 

<div class="bg-cover bg-center ..." style="background-image: url(...)"></div>

Reply:
<div class="bg-cover bg-center ..." style="background-image: url(...)"></div>

Reply:
or div class="bg-[url('/img.jpg')]"

Reply:
> or div class="bg-[url('/img.jpg')]"

thank you! On it!